io.github.quodlibet.QuodLibet.yaml

```diff
@@ -15,6 +15,12 @@ finish-args:
   - "--socket=pulseaudio"
   - "--filesystem=host"
 
+  # some embedded covers and the fifo is still shared through /tmp
+  - "--filesystem=/tmp"
+
+  # bookmarks in the filesystem browser
+  - "--filesystem=xdg-config/gtk-3.0:ro"
```

Comment:
The correct solution is to use GtkFileChooserNative, this is just a workaround that nothing else on Flathub uses.

Reply:
We use GtkFileChooserNative, but we have our own filesystem browser widget